examples/esp/.cargo/config.toml

```diff
@@ -1,21 +1,19 @@
 [target.riscv32imac-unknown-none-elf]
 runner = "probe-rs run --chip esp32c6"
```

Comment:
we should use `espflash` as the runner since we are using `esp-println`

Reply:
A copy-paste mistake from an individual who never uses `cargo run` on embedded. Something like this, or...?
`runner = "espflash flash --chip esp32c6 --monitor --baud 1500000"`

Reply:
haha - fair

just `runner = "espflash flash --monitor"`